src/components/MenuPageContainer/styles.js

```diff
@@ -0,0 +1,106 @@
+export const containerMenu = {
+  textAlign: "center",
+  minWidth: 1200,
+  minHeight: 1056,
+  display: "flex",
+  alignItems: "center",
+  backgroundColor: "#F5FBFC",
+  color: "black",
+  flexDirection: "column",
+  marginTop: 140,
+};
+
+export const phoneTooltip = {
+  position: "relative",
+  cursor: "pointer",
+  textDecoration: "none",
+
+  color: "#35B8BE",
+  textAlign: "center",
+  fontFamily: "Inter",
+  fontSize: 16,
+  fontStyle: "normal",
+  fontWeight: 400,
+  letterSpacing: 0.36,
+};
+
+export const phoneTooltipHover = {
+  content: '"123-456-7890"',
+  position: "absolute",
+  bottom: "100%",
+  left: "50%",
+  transform: "translateX(-50%)",
+  backgroundColor: "#333",
+  color: "#fff",
+  padding: "5px 10px",
+  borderRadius: "4px",
+  whiteSpace: "nowrap",
+  marginBottom: "5px",
+  zIndex: 1,
+};
+export const menuPageFilterButtonsContainer = {
+  marginTop: 50,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  width: 1200,
+  height: 52,
+};
+
+export const menuPageFilterButtons = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "space-between",
+  width: 456,
+  height: 52,
+};
+
+export const menuPageButtons = {
+  width: 132,
+  height: 52,
+  borderRadius: 6,
+  border: "1px solid rgba(97, 114, 131, 0.20)",
+
+  color: "#222",
+  fontFamily: "Inter",
+  fontSize: 16,
+  fontStyle: "normal",
+  fontWeight: 400,
+  backgroundColor: "#FFF",
+};
+
+export const menuDescribe = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  flexDirection: "column",
+  width: 670,
+  height: 159,
+};
+
+export const hMenu = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  margin: 0,
+  width: 442,
+  color: "#35B8BE",
+  textAlign: "center",
+  fontFamily: "Inter",
+  fontSize: 50,
+  fontStyle: "normal",
```

Comment:
It is a default font-style, so, we don't have to type it manually if there is no necessity